tests/Fixtures/TestUser.php

```diff
@@ -10,6 +10,9 @@
 
 class TestUser extends Model implements Authenticatable, FilamentUser
 {
+    /**
```

Comment:
Laravel 11 added some new phpstan annotations. Because of this we now need 2 `phpstan.neon` files for the github flows since the requirements differ now between v11 and v10.